plugins/node/instrumentation-runtime/src/instrumentation.ts

```diff
@@ -0,0 +1,266 @@
+      GaugeNames.NODE_EVENT_LOOP_DELAY_P99,
+      {
+        description: 'The 99th percentile of the recorded event loop delays.',
+        unit: 'ms',
+      }
+    );
+
+    // Event Loop Delay metrics
+    this.meter.addBatchObservableCallback(
+      async observable => {
+        const attributes = this._getCustomMetricAttributes();
+
+        const startTime = hrTime();
+        // Waits for all existing and chained promises to be resolved:
+        await new Promise(setImmediate);
+        const endTime = hrTime();
+
+        // Calculate the delta between scheduling a promise and the event loop handling it.
+        // This is the delay of the event loop.
+        const duration = hrTimeToMilliseconds(
+          hrTimeDuration(startTime, endTime)
+        );
+        observable.observe(eventLoopDelayGauge, duration, attributes);
+
+        function nanoSecondToMilliSecond(nanoSecond: number) {
+          return nanoSecond / 1e6;
+        }
+
+        observable.observe(
+          eventLoopDelayMinGauge,
+          nanoSecondToMilliSecond(this.eventLoopDelayHistogram.min),
+          attributes
+        );
+        observable.observe(
+          eventLoopDelayMaxGauge,
+          nanoSecondToMilliSecond(this.eventLoopDelayHistogram.max),
+          attributes
+        );
+
+        observable.observe(
+          eventLoopDelayMeanGauge,
+          nanoSecondToMilliSecond(this.eventLoopDelayHistogram.mean),
+          attributes
+        );
+
+        observable.observe(
+          eventLoopDelayStddevGauge,
+          nanoSecondToMilliSecond(this.eventLoopDelayHistogram.stddev),
+          attributes
+        );
+
+        observable.observe(
+          eventLoopDelayP50Gauge,
+          nanoSecondToMilliSecond(this.eventLoopDelayHistogram.percentile(50)),
+          attributes
+        );
+
+        observable.observe(
+          eventLoopDelayP95Gauge,
+          nanoSecondToMilliSecond(this.eventLoopDelayHistogram.percentile(95)),
+          attributes
+        );
+
+        observable.observe(
+          eventLoopDelayP99Gauge,
+          nanoSecondToMilliSecond(this.eventLoopDelayHistogram.percentile(99)),
+          attributes
+        );
+      },
+      [
+        eventLoopDelayGauge,
+        eventLoopDelayMinGauge,
+        eventLoopDelayMaxGauge,
+        eventLoopDelayMeanGauge,
+        eventLoopDelayStddevGauge,
+        eventLoopDelayP50Gauge,
+        eventLoopDelayP95Gauge,
+        eventLoopDelayP99Gauge,
+      ]
+    );
+
+    const loopUtilizationGauge = this.meter.createObservableGauge(
+      GaugeNames.NODE_EVENT_LOOP_UTILIZATION,
+      {
+        description: 'The percentage utilization of the event loop.',
+        unit: 'percent',
+      }
+    );
+
+    const loopIdleGauge = this.meter.createObservableGauge(
+      GaugeNames.NODE_EVENT_LOOP_UTILIZATION_IDLE,
+      {
+        description: 'The idle time utilization of event loop.',
+        unit: 'ms',
```

Comment:
I think we may need to use seconds for this instead: https://github.com/open-telemetry/semantic-conventions/blob/main/docs/general/metrics.md#instrument-units